.github/workflows/swiftpm.yml

```diff
@@ -0,0 +1,38 @@
+name: SwiftPM Build CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  swiftpm_macos_test:
+    name: Build with SwiftPM on macOS
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-13, macos-14]
+        xcode-version: ["15.2"]
+        target: ["Life", "SwiftBreak"]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Xcode
+        uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: ${{ matrix.xcode-version }}
+      - uses: SwiftyLab/setup-swift@latest
+        with:
+          development: true
+          swift-version: latest
```

Comment:
Do you prefer to use the latest nightly toolchain or use a fixed nightly version here?